kornia/geometry/transform/homography_warper.py

```diff
@@ -21,6 +21,13 @@
 ]
 
 
+def _kornia_inverse(input: torch.Tensor) -> torch.Tensor:
+    dtype: torch.dtype = input.dtype
+    if not dtype in (torch.float32, torch.float64,):
+        dtype = torch.float32
```

Comment:
probably put it into ```kornia.utils``` than duplicated twice in files.

Reply:
add some explaination notes.

Reply:
this was just a quick test - we have to think about how to handle precision for some regression tests where we have harcoded numbers